src/actions/index.js

```diff
@@ -1,4 +1,6 @@
 import axios from 'axios';
+import { instanceLocator, tokenUrl } from '../components/direct_message_config';
```

Comment:
are you using these in here? also could you take this out of the components and put it in the constants file pls? thanks

Reply:
Moved :)